### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Lehren**
Add jsdoc rules, change docs to fit those rules. And remove a leftover unused property.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -14,3 +14,13 @@ plugins:
 rules:
   no-console: 0  # This project purposely uses console.
   prefer-reflect: 0
+  jsdoc/check-param-names: 1
```
Comment (kvdb) [inline .eslintrc]: you might want to add these rules to https://github.com/dcentralize/eslint-config-d-centralize ?

File: test/audio-recorderSpec.js [hunk 2]
```diff
@@ -29,7 +28,6 @@ describe('Audio recorder', () => {
     const recorder = new AudioRecorder();
     expect(recorder.settings).toEqual({});
     expect(recorder.userMediaApproval).toBeFalsy();
-    expect(recorder.events).toEqual({});
```
Comment (kvdb) [inline test/audio-recorderSpec.js]: unrelated

File: .eslintrc (hunk 1 above)
Comment (Lehren) [inline .eslintrc]: They only work with the appropiate JSdoc-eslint plugin installed so it's quite project specific.

https://github.com/gajus/eslint-plugin-jsdoc

File: test/audio-recorderSpec.js (hunk 2 above)
Comment (Lehren) [inline test/audio-recorderSpec.js]: Removed.
